tapdb/sqlc/queries/universe.sql

```diff
@@ -260,3 +260,33 @@ WHERE event_type IN ('SYNC', 'NEW_PROOF') AND
       event_timestamp >= @start_time AND event_timestamp <= @end_time
 GROUP BY day
 ORDER BY day;
+
+-- name: SetFederationGlobalSyncConfig :exec
+INSERT INTO federation_global_sync_config (
+    proof_type, allow_sync_insert, allow_sync_export
+)
+VALUES (@proof_type, @allow_sync_insert, @allow_sync_export)
+ON CONFLICT(proof_type)
```

Comment:
Since we're overwriting on conflict, I think this should be called `UpsertFederationGlobalSyncConfig`. Would also better match the name used for the universe specific one below.